app/views/burritos/_burrito.html.erb

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
You shouldn't need the `<html>`, `<head>`, or `<body>`, element in any View template that is not a layout View template. The default layout is `application.html.erb`— and that all the required HTML elements you need. It also looks like you load fontawesome in that file as well, which should make it possible to remove the `<link>` here.